README.md

```diff
@@ -5,22 +5,22 @@ Bitcoin Core integration/staging tree
 
 https://www.bitcoin.org
 
-What is Bitcoin?
+What is Bitcoin Classic?
 ----------------
 
-Bitcoin is an experimental new digital currency that enables instant payments to
+Bitcoin is an experimental new digital forkcoin that enables instant payments to
 anyone, anywhere in the world. Bitcoin uses peer-to-peer technology to operate
-with no central authority: managing transactions and issuing money are carried
+with central authority: managing transactions and issuing money are carried
 out collectively by the network. Bitcoin Core is the name of open source
 software which enables the use of this currency.
 
```

Comment:
Should we also explain classic *democratic* decision making https://github.com/bitcoinclassic/bitcoinclassic/pull/14 *governance model* here?